### PR TITLE
Facepile resize group

### PR DESCRIPTION
**Pull request by mattlind**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [ ] Include a change request file using `$ npm run change`

I'll add this after initial review and changes/testing/cleanup are done.

#### Description of changes

Had some time to try a new iteration of ResizableFacepile.  Not the final but I wanted to get a quick code review.  I have a bit of cleanup and some experiments to run to see how it looks in a real application with live resizing of multiple Facepile components.

#### Focus areas to test

Resizing on window size change.  Resizing when multiple Facepiles exist on the page.

### CODE REVIEW

File: packages/office-ui-fabric-react/src/components/Facepile/Facepile.scss [hunk 1]
```diff
@@ -18,7 +18,7 @@ $spacingAroundItemButton: 2px;
 .members {
   display: flex;
   flex-direction: row;
-  flex-wrap: wrap;
+  flex-wrap: nowrap;
```
Comment (phkuo) [inline packages/office-ui-fabric-react/src/components/Facepile/Facepile.scss]: this is default, just remove it

File: packages/office-ui-fabric-react/src/components/Facepile/Facepile.scss [hunk 2]
```diff
@@ -18,7 +18,7 @@ $spacingAroundItemButton: 2px;
 .members {
   display: flex;
   flex-direction: row;
```
Comment (phkuo) [inline packages/office-ui-fabric-react/src/components/Facepile/Facepile.scss]: this is also default, remove it

**Comment by mattlind** [comment]
Holding on this until https://github.com/OfficeDev/office-ui-fabric-react/pull/4096 is in.  I haven't forgotten about this.

**Comment by dzearing** [comment]
@mattlind the other pr is now merged.

I am going to close as this PR is ages old and we are trying to aggressively reduce/resolve all open PRs. When you have time, can you please re-open when you have resolved the conflicts and feedback?

Thank you!

**Pull request closed without merging**